Controller/AdminSecurityController.php

```diff
@@ -12,29 +12,29 @@
 namespace Sonata\UserBundle\Controller;
 
 use FOS\UserBundle\Model\UserInterface;
-use Symfony\Component\DependencyInjection\ContainerAware;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\RedirectResponse;
```

Comment:
It's used here: https://github.com/sonata-project/SonataUserBundle/pull/641/files#diff-b773cf909994196ea448ba8b717c1c55R24

BTW, if it's not use, php-cs-fixer would detect it. :+1: